astropy/visualization/wcsaxes/ticklabels.py

```diff
@@ -128,23 +139,23 @@ def draw(self, renderer, bboxes, ticklabels_bbox):
                     if np.abs(self.angle[axis][i]) < 45.:
                         ha = 'right'
                         va = 'bottom'
-                        dx = - text_size * 0.5
+                        dx = - pad
```

Comment:
Why do you have a space after the minus sign (just curious)?  (and also in line 148).

Reply:
Not sure, will fix!

Reply:
and line 153 too...